### PR TITLE
[ticket/14416] footer nav links have incorrect tooltip styling applied

### PR DESCRIPTION
**Pull request by RMcGirr83**
https://tracker.phpbb.com/browse/PHPBB3-14416

PHPBB3-14416

### CODE REVIEW

**Comment by RMcGirr83** [comment]
!set Styles

**Comment by Nicofuma** [comment]
Just removing the spaces should fix the issue?

File: phpBB/styles/prosilver/template/navbar_footer.html [hunk 1]
```diff
@@ -4,18 +4,10 @@
...
-				</span>
+				<span class="crumb"><a href="{U_SITE_HOME}" data-navbar-reference="home"><i class="icon fa-home fa-fw" aria-hidden="true"></i><span>{L_SITE_HOME}</span></a></span>
 			<!-- ENDIF -->
 			<!-- EVENT overall_footer_breadcrumb_prepend -->
-			<span class="crumb">
-				<a href="{U_INDEX}" data-navbar-reference="index">
-					<!-- IF not U_SITE_HOME --><i class="icon fa-home fa-fw" aria-hidden="true"></i><!-- ENDIF --><span>{L_INDEX}</span>
-				</a>
-			</span>
+				<span class="crumb"><a href="{U_INDEX}" accesskey="h" data-navbar-reference="index"><!-- IF not U_SITE_HOME --><i class="icon fa-home fa-fw"></i><!-- ENDIF --><span>{L_INDEX}</span></a></span>
```
Comment (Nicofuma) [inline phpBB/styles/prosilver/template/navbar_footer.html]: `accesskey="h"` on the `<a>` tag is new and `aria-hidden="true"` is on the `li` tag

Comment (Elsensee) [inline phpBB/styles/prosilver/template/navbar_footer.html]: Indeed, I remember that more than one accesskey attribute pointing to the same key will break that key in firefox.

Comment (RMcGirr83) [inline phpBB/styles/prosilver/template/navbar_footer.html]: Oddly enough, yes

**Comment by Nicofuma** [comment]
in this case, instead of inlining everything, you should wrap it by the spaceless tag `{% spaceless %} {% endspaceless %}`

**Comment by Nicofuma** [comment]
http://twig.sensiolabs.org/doc/tags/spaceless.html

**Comment by hanakin** [comment]
wow twig made it into 3.2 :+1:  lol

**Comment by Elsensee** [comment]
@hanakin We have a whole twig template already in 3.2. It's for the feeds and in styles/all/template ;)

**Pull request merged**